### DISCUSSION TITLE
Interesting bug

### DISCUSSION THREAD

**Discussion by MikleBas** [wavelog/wavelog#1366] [General]
Actually, everything is visible on the screenshots, it is unclear where the program pulls this ill-fated email from? :)

![1](https://github.com/user-attachments/assets/f654e637-1bc3-4593-87be-cc152993dca7)
![2](https://github.com/user-attachments/assets/96eeca6a-5bba-42f2-84f3-6ea98c798af8)
![3](https://github.com/user-attachments/assets/59699ace-fcee-4efd-88a6-8b5502d8b8d0)

**Comment by int2001** [comment]
Confirming bug.

investigating

**Comment by int2001** [comment]
Bug should be resolved by #1367
